app/controllers/vm_common.rb

```diff
@@ -558,6 +558,18 @@ def snap
   end
   alias_method :vm_snapshot_add, :snap
 
+  def render_missing_field(session, missing_field_name)
+    add_flash(_("#{missing_field_name} is required"), :error)
+    @in_a_form = true
+    drop_breadcrumb(:name => _("Snapshot VM '#{@record.name}'"), :url => "/vm_common/snap")
```

Comment:
this won't work with i18n.
please use the format in the original code
See http://manageiq.org/docs/guides/i18n and search for 'string interpolation'